layouts/partials/social.html

```diff
@@ -23,6 +23,15 @@
     </a>
     {{ end }}
 
+    {{ if isset .Site.Params "mastodon" }}
+	<!-- Icon from https://icons.getbootstrap.com/icons/mastodon/ -->
+    <a class="social-icon" href="{{ .Site.Params.Mastodon }}" target="_blank" rel="noopener" title="Mastodon">
```

Comment:
To allow Mastodon to mark the link as verified

```suggestion
    <a class="social-icon" href="{{ .Site.Params.Mastodon }}" target="_blank" rel="me" title="Mastodon">
```